### PR TITLE
Improved backup docs https://github.com/apache/jena/issues/1500

### PR DESCRIPTION
**Pull request by ieugen**
Signed-off-by: Eugen Stan <ieugne@apache.org>

### CODE REVIEW

File: source/documentation/tdb2/tdb2_admin.md [hunk 1]
```diff
@@ -67,3 +67,13 @@ continue to be serviced.
 
 There is also a command line tool `tdb2.tdbbackup` to run the
 backup process on a database not in use.
+
```
Comment (afs) [inline source/documentation/tdb2/tdb2_admin.md:70]: `tdb2.tdbbackup` has had this feature for a long time. `tdb2.tdbbackup` isn't really very important. Nowadays, tarsnaction handling applies to all databases so special casing it isn't necessary. TBH It should probably be removed.

The recent PR put the same functionality into Fuseki - they don't share code.

The Fuseki code isworks with any database type, integrates into the Fuseki lifecycle and Fuseki logging.

Could you put this change in the Fuseki docs please?

**Review by afs — CHANGES_REQUESTED** [review]

File: source/documentation/tdb2/tdb2_admin.md (hunk 1 above)
Comment (ieugen) [reply source/documentation/tdb2/tdb2_admin.md:70]: I am a bit confused. 
You want me to drop this PR and put the same text in fuseki docs here: https://github.com/apache/jena-site/blob/main/source/documentation/fuseki2/fuseki-server-protocol.md ?

**Comment by ieugen** [comment]
done.

**Review by afs — APPROVED** [review]

**Comment by afs** [comment]
Thanks! Looks good.

Because this refers to 4.7.0, it won't get merged to main just yet.

(separately we're discussing changing the publication pipeline to have a branch for future changes; we only have a per-PR staging at the moment).

**Comment by ieugen** [comment]
> (separately we're discussing changing the publication pipeline to have a branch for future changes; we only have a per-PR staging at the moment). 

This sounds great. Thanks.
p.s. I did not have time to work on the backup stuff, plan to do that this year.

**Pull request merged**